test/gtest/uct/ib/test_ib_pkey.cc

```diff
@@ -106,11 +104,9 @@ class test_uct_ib_pkey : public test_uct_ib_with_specific_port {
                 supported_pkeys_idx.push_back(table_idx);
             }
 
-            supported_pkey_pairs = ucs::make_pairs(supported_pkeys);
-            supported_pkey_idx_pairs = ucs::make_pairs(supported_pkeys_idx);
-
-            result = std::make_pair(supported_pkey_pairs,
-                                    supported_pkey_idx_pairs);
+            // rely on RVO for the created pairs and the result
```

Comment:
Are you sure RVO works here, considering `static`?

Reply:
added std::move

Reply:
maybe it should work since the object which is going to be optimized is not a `static` variable